packages/admin/admin-rte/src/core/Controls/LinkControls.tsx

```diff
@@ -1,44 +1,54 @@
-import { ButtonGroup, ComponentsOverrides, Theme } from "@mui/material";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import { ThemedComponentBaseProps } from "@comet/admin";
+import { ButtonGroup, ComponentsOverrides, css, styled, Theme, useThemeProps } from "@mui/material";
 import * as React from "react";
 
 import LinkToolbarButton from "../extension/Link/ToolbarButton";
 import LinksRemoveToolbarButton from "../extension/LinksRemove/ToolbarButton";
 import { IControlProps } from "../types";
 
-function LinkControls(p: IControlProps & WithStyles<typeof styles>) {
+export type RteLinkControlsProps = ThemedComponentBaseProps<{ root: typeof ButtonGroup; item: "div" }>;
+
+function StyledLinkControls(p: IControlProps, inProps: RteLinkControlsProps) {
```

Comment:
There can only be one props argument...

Should `RteLinkControlsProps` extend from `IControlProps`? 
Or should `IControlProps` extend from `ThemedComponentBaseProps` instead of adding a second props type?